ui/src/pages/backoffice/Series/SeriesDetails/SeriesDetails.js

```diff
@@ -41,10 +41,18 @@ export default class SeriesDetails extends Component {
 
   seriesPanels = () => {
     const { data } = this.props;
+    let literatureNum = 0
```

Comment:
If I am not wrong, you can have only one or the other, so what about checking both and summing? One should be always 0...

You can also use (to be tested!!!) [lodash get]() here:
```
const serialsCount = _get(data, 'metadata.relations_metadata.serial', []).length
const monographsCount = _get(data, 'metadata.relations_metadata.multipart_monograph', []).length
const documentsCount = serialsCount + monographsCount
```

and then below:
```
title: 'Literatures in this series (' + documentsCount + ')',
```

WDYT?